utilities/blob_db/blob_db.h

```diff
@@ -78,6 +78,13 @@ struct BlobDBOptions {
   // blob files will be cleanup based on TTL.
   bool enable_garbage_collection = false;
 
+  // Time interval to trigger garbage collection, in seconds.
+  uint64_t garbage_collection_interval_secs = 60;
+
+  // If garbage collection is enabled, blob files with deleted size no less
+  // than this ratio will become candidates to be cleanup.
+  double garbage_collection_deletion_size_threshold = 0.75;
```

Comment:
I like this change. I always wanted to move these to the public header file (blob_db.h) and make them configurable instead of being constants.